org.osgi.service.jakartaws/src/org/osgi/service/jakartaws/runtime/package-info.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0 
+ *******************************************************************************/
+
+/**
+ * Jakarta Web Services Runtime Package Version 1.0.
+ * <p>
+ * Bundles wishing to use this package must list the package in the
+ * Import-Package header of the bundle's manifest. This package has two types of
+ * users: the consumers that use the API in this package and the providers that
+ * implement the API in this package.
+ * <p>
+ * Example import for consumers using the API in this package:
+ * <p>
+ * {@code  Import-Package: org.osgi.service.jakartaws.runtime; version="[1.0,2.0)"}
+ * <p>
+ * Example import for providers implementing the API in this package:
+ * <p>
+ * {@code  Import-Package: org.osgi.service.jakartaws.runtime; version="[1.0,1.1)"}
+ * 
+ * @author $Id$
+ */
+
+@Version(SOAP_SPECIFICATION_VERSION + ".0")
```

Comment:
The `.0` is unnecessary.

Reply:
I have taken this pattern from the REST implementation.
My guess is / was that the idea is that the spec version always stays the same (1.0) and the last part is possibly incremented (e.g. SOAP_SPECIFICATION_VERSION + ".1") in case some API enhancements are performed that require an update to the package but not to the specification version.